draft/draft-ietf-rtgwg-qos-model.xml

```diff
@@ -297,17 +293,18 @@
     </section>
 
     <section title=" Modules Tree Structure">
-      <t>This document defines seven YANG modules - four QoS base
-      modules, a scheduler policy module, a queuing policy module and
-      one DiffServ module.</t>
-
-      <t>ietf-traffic-policy module contains list of classifier entries
-      identified by a classifier entry name. Each entry MAY contain a list of
-      filter entries.  When no filter entry is present in a classifier
-      entry, it matches all traffic.</t>
-
-      <t>ietf-traffic-policy module also contains list of policy objects identified
-      by a policy name and policy type which MUST be provided. With different
+      <t>This document defines a base traffic policy module, a QoS
```

Comment:
I would suggest that we give names to the modules, and refer to them by their name. For example, you could call ietf-traffic-policy module as the Base Traffic Policy Module, and replace all references to ietf-traffic-policy module as Base Traffic Policy Module.

Reply:
then, somewhere we have to define atleast these three modules:

ietff-traffic-policy:         Base Policy Module
ietf-diffserv-policy:       Diffserv Module
ietf-qos-action:             QoS Action Module

Reply:
I have fixed in new PR for Issue #53